src/libraries/System.Private.CoreLib/src/System/Half.cs

```diff
@@ -266,7 +266,7 @@ public static Half Parse(string s, NumberStyles style)
         public static Half Parse(string s, IFormatProvider? provider)
         {
             if (s == null) ThrowHelper.ThrowArgumentNullException(ExceptionArgument.s);
-            return Number.ParseHalf(s, NumberStyles.Float | NumberStyles.AllowThousands, NumberFormatInfo.GetInstance(provider));
+            return Number.ParseHalf(s, DefaultParseStyle, NumberFormatInfo.GetInstance(provider));
```

Comment:
Noting that this and above are the only changes to non preview code and they're a no-op. (It might have been nice if we used DefaultParseStyle in Float and Double or not at all, so they look the same)